robottelo/vm_capsule.py

```diff
@@ -127,38 +113,24 @@ def capsule_location_ids(self):
     def capsule_organization_ids(self):
         return self._capsule_organization_ids
 
-    def _capsule_setup_ddns(self):
```

Comment:
yeah. this is not required anymore.
also, that approach was really buggy as if the vm stayed up and runnign in place, it was updating the ddns record (every 5 minutes), runing other tests, since dns was pointing to an invalid machine.

- we only use capsules here to test the pulp node features, so we don't really need dhcp smart proxy.
- also, we are now able to run the tests in parralel, since the VM's do not interfere with each other (anyway, it's not worth starting to execute them in parralel due to relatively heavy setup)

Reply:
@rplevka are you sure that sat will resolve the VM host name properly?

Reply:
@rplevka If we are not going to setup a ddns , we have to remote that from the settings and properties file also.

Reply:
@rplevka think that we need a fully functional capsule vm for any case not only for the tests working today but for possible future tests also.

Reply:
> are you sure that sat will resolve the VM host name properly?

Yes.
> If we are not going to setup a ddns , we have to remote that from the settings and properties file also.

if you mean "remove" then ok, I will do the cleanup

> think that we need a fully functional capsule vm for any case not only for the tests working today but for possible future tests also.

ddns would not help us here either. For providing dhcp, dns, tftp, etc features we need the capsule to control a provisioning subnet. I believe we can do this later using the vlans same as we do for satellite provisioning. However, i strongly believe we will move towards some real lab.